kit/Kit.cpp

```diff
@@ -1711,23 +1711,20 @@ void Document::notifyViewInfo()
                         Parser parser;
                         Poco::Dynamic::Var var = parser.parse(userPrivateInfo);
                         Object::Ptr userPrivateInfoObj = var.extract<Object::Ptr>();
-                        std::string signatureCert;
-                        JsonUtil::findJSONValue(userPrivateInfoObj, "SignatureCert", signatureCert);
-                        if (!signatureCert.empty())
+                        std::initializer_list<std::string> keys = {
+                            "SignatureCert",
+                            "SignatureKey",
+                            "SignatureCa",
+                            "ESignatureSecret",
```

Comment:
This is just adding the ESignatureSecret as a key to be filtered + refactor to avoid further copy & paste.